README.md

```diff
@@ -1 +1,29 @@
 # sync-templates
+
+[![Continuous Integration](https://github.com/fiberplane/sync-templates/actions/workflows/ci.yml/badge.svg)](https://github.com/fiberplane/sync-templates/actions/workflows/ci.yml)
+
+The `fiberplane/sync-templates` action is a utility that syncs your Fiberplane Templates from a designated directory in your repository with your Fiberplane Workspace.
+
+## Usage
+This action can be run on `ubuntu-latest` and `macos-latest` GitHub Actions runners.
+
+A minimum working example of the action:
+```yaml
+steps:
+- uses: fiberplane/sync-templates@v1
+  with:
+    api-token: ${{ secrets.FP_TOKEN }} # it is best practice to keep your secrets in GitHub Secrets
+    workspace-id: ${{ secrets.FP_WORKSPACE_ID }}
+```
+
+The `sync-templates` action accepts the following inputs:
+- `api-token` (**required**) - the Fiberplane API token used to access the workspace
+- `workspace-id` (**required**) - the ID of the workspace where the Templates should be sync'ed to
+- `templates-directory` (optional) - directory where valid Templates `*.jsonnet` files are located, default: `.fiberplane/templates/`
```

Comment:
I was thinking, maybe we should make the default without a `.` prefix. So something like `fiberplane/templates/`

To me a hidden directory is something that is secondary to the repo. For example having your own code is the primary use case of a application repo, but it might also contain a `.github` directory containing workflows. In the case of this action it will be run on a infrastructure like repo, where its main purpose is to store the templates.

Reply:
I don't entirely agree here. I was actually thinking that the primary way this would be used would be per team / per project basis (kinda like hooks or workflows). Teams and/or projects might want to have project-specific templates attached in which case they become like a workflow associated with the repository code. Hence the `.fiberplane/` prefix

Reply:
> I don't entirely agree here. I was actually thinking that the primary way this would be used would be per team / per project basis (kinda like hooks or workflows). Teams and/or projects might want to have project-specific templates attached in which case they become like a workflow associated with the repository code. Hence the `.fiberplane/` prefix

This kinda goes against what is planned and what you mentioned earlier, that if a template is not defined in the directory then it should be deleted. Ie. the directory should match the templates that are created 1to1. If the idea is to have this in multiple directories than that workflow is not possible.

Reply:
yeah that's a good point. I'm personally not yet entirely settled how the `delete` workflow should work but this is a good gotcha to keep in mind when we're discussing it